app/assets/stylesheets/homepage.css.scss

```diff
@@ -16,14 +16,39 @@
     font-weight: 200;
 }
 
-.nav-bar-nk {
+#logo {
+    text-decoration: none;
+    margin-left: -70px ;
+}
+
+.nav-bar-link {
     color: white;
-    font-size: 20px;
+    font-size: 1.1em;
+    text-decoration: none;
+    height: 76px;
+    display: block;
+    padding: 8px 12px 0 12px;
+}
+
+.nav-bar-link:hover {
+  background-color: rgba(0,0,0,.4);
 }
 
-#username {
+.username-link {
     color: white;
-    font-size: 20px;
+    font-size: 1.45em;
+    text-decoration: none;
+    height: 76px;
+    display: block;
+    padding: 12px 0px 0 12px;
+}
+
+.username-link:hover {
+  background-color: rgba(0,0,0,.4);
+}
+
+#nav-bar .mdl-layout__header-row .mdl-navigation__link {
```

Comment:
Avoid using id selectors<br>Merge rule `#nav-bar .mdl-layout__header-row .mdl-navigation__link` with rule on line 6<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `mdl-layout__header-row` should be written in lowercase with hyphens<br>Selector `mdl-navigation__link` should be written in lowercase with hyphens